src/python/pants/init/extension_loader.py

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
There's a lot of noise to get the formatting to look as intended, particularly to get hard wrapping working correctly with new lines and indentation. While this is complex, I argue it's worth it for a much nicer error message.